roster2/src/webapp/js/roster.js

```diff
@@ -30,6 +30,15 @@
 	});
     
 	roster.i18n = $.i18n.map;
+
+    roster.i18n.months = roster.i18n.months.split(',');
+
+    /*
```

Comment:
It's not coffee, it's the loveliness of ECMA 5. No, it shouldn't be there though. I'll remove it forthwith :)